bundles/org.openhab.binding.velux/README.md

```diff
@@ -238,7 +238,7 @@ The supported Channel and its associated channel type is shown below.
 Unfortunately Velux has decided to closely integrate the rain sensor into the window device.
 The rain sensor is therefore not displayed in the device list.
 On the other hand, the 'limitMinimum' channel of a roof window provides information about rainy weather:
-If it is set internally by the Velux control unit to a value other than zero, it rains. (Joke!!)
+If it is set internally by the Velux control unit to a value other than zero, it rains.
```

Comment:
@Chiuaua79 response..

> Hi Andrew,
>
> English is not my mother tongue, indeed. Dutch t.b.h. I doubt if many other non-native English speakers get the pun. I suggest to rephrase it to prevent a “joke” like it is in the current version, confuses all the users to understand it is a non-factual paragraph and limitMinimum is not related at all to the rain sensors of the window.
>
> Thanks for reviewing. :)
>
> Krgds
> Cor Hoogendoorn

Reply:
I did not write that text; actually @gs4711 wrote it; and I added the 'Joke' suffix because I was not sure what he intended; -- @gs4711 maybe you can clarify it now?